djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
+
+    if version == 'latest':
```

Comment:
["Explicit is better than implicit."](https://www.python.org/dev/peps/pep-0020/). I like the idea of having a settings variable to change the supported version (along with the warning you added).